include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -168,15 +147,8 @@
 
 // Should be defined to 1 for environments with a vendor implementation of C++17 execution policies
 #define _ONEDPL_CPP17_EXECUTION_POLICIES_PRESENT                                                                       \
-    (_MSC_VER >= 1912 && _MSVC_LANG >= 201703L) ||                                                                     \
-        (_GLIBCXX_RELEASE >= 9 && __GLIBCXX__ >= 20190503 && __cplusplus >= 201703L)
-#define _ONEDPL_CPP14_2RANGE_MISMATCH_EQUAL_PRESENT                                                                    \
-    (_MSC_VER >= 1900 || __cplusplus >= 201300L || __cpp_lib_robust_nonmodifying_seq_ops == 201304)
-#define _ONEDPL_CPP14_MAKE_REVERSE_ITERATOR_PRESENT                                                                    \
-    (_MSC_VER >= 1900 || __cplusplus >= 201402L || __cpp_lib_make_reverse_iterator == 201402)
-#define _ONEDPL_CPP14_INTEGER_SEQUENCE_PRESENT (_MSC_VER >= 1900 || __cplusplus >= 201402L)
-#define _ONEDPL_CPP14_VARIABLE_TEMPLATES_PRESENT                                                                       \
-    (!__INTEL_COMPILER || __INTEL_COMPILER >= 1700) && (_MSC_FULL_VER >= 190023918 || __cplusplus >= 201402L)
+    (_MSC_VER >= 1912 && _MSVC_LANG >= 201703L) || (_GLIBCXX_RELEASE >= 9 && __GLIBCXX__ >= 20190503)
+#define _ONEDPL_CPP14_VARIABLE_TEMPLATES_PRESENT (!__INTEL_COMPILER || __INTEL_COMPILER >= 1700)
 #define _ONEDPL_CPP11_IS_TRIVIALLY_COPY_ASSIGNABLE_PRESENT                                                             \
```

Comment:
Do we really need to keep it? Looks that we should it remove to because it is for  "// Required to support GNU libstdc++ below 5.x", which is very old version and I pretty doubt that it works with C++17 compiler,

Reply:
Fixed: define removed.